packages/vitest/src/types/config.ts

```diff
@@ -344,6 +344,15 @@ export interface InlineConfig {
    * Return `false` to ignore the log.
    */
   onConsoleLog?: (log: string, type: 'stdout' | 'stderr') => false | void
+
+  /**
+   * Should css be processed.
+   *
+   * If regexp is provided, it will transform matched files.
+   *
+   * @default [/\.module\./]
+   */
+  processCss?: boolean | RegExp[]
```

Comment:
This looks like a great DX improvement. About naming, given that there is already a `css` option in Vite, maybe this new option could be aligned with it:
```
  css: {
    include: RegExp | RegExp[], // @default [/\.module\./]
    exclude: RegExp | RegExp[]
  }
```
Removing the `process` in the name could be helpful if there are other CSS options added later. Or maybe `css: boolean | { include, exclude }` if you still want users to use `css: true`

Reply:
I think if we want to add more features to css, they should be added to Vite’s options :)

About the object - I think yes, we can align it with other similar options.